user/templates/profile/index.html

```
@@ -175,49 +175,50 @@ <h3 class="h-3 text-gray-700 dark:text-gray-200 mb-4 md:mb-8">{% if contact_prof
-                        <select name="privacy" id="privacy"
-                            class="w-full mt-1 px-3 py-2 border border-gray-300 dark:border-gray-600 bg-white dark:bg-gray-900 text-gray-900 dark:text-gray-100 rounded-md shadow-sm focus:ring focus:ring-blue-300">
-                            {% for val, label in privacy_choices %}
-                                <option value="{{ val }}" {% if contact.privacy == val %}selected{% endif %}>{{ label }}</option>
-                            {% endfor %}
-                        </select>
+                        <div>
+                            <label for="privacy" class="block text-sm text-gray-700 dark:text-gray-300">{% translate 'Privacy' %}</label>
+                            <select name="privacy" id="privacy"
+                                class="w-full mt-1 px-3 py-2 border border-gray-300 dark:border-gray-600 bg-white dark:bg-gray-900 text-gray-900 dark:text-gray-100 rounded-md shadow-sm focus:ring focus:ring-blue-300">
+                                {% for val, label in privacy_choices %}
+                                    <option value="{{ val }}" {% if contact.privacy == val %}selected{% endif %}>{{ label }}</option>
```

HTMLHint (1.5.0)

[error] The value of attribute [ contact.privacy ] must be in double quotes.

(attr-value-double-quotes)